UefiPayloadPkg/SPI/pci_mmio_cfg.h

```diff
@@ -0,0 +1,50 @@
+// #ifndef PCI_MMIO_CFG_H
```

Comment:
This whole file contains comments... Remove it @IgorBagnucki

Reply:
Deleted it